### PR TITLE
Expands reviews module with additional components

### PR DESCRIPTION
**Pull request by jayrbarr**

### CODE REVIEW

**Comment by jayrbarr** [comment]
@timothydadams @JasperC1999 This has some of my proxy config and other things in it.

File: client/components/App.jsx [hunk 1]
```diff
@@ -16,7 +16,18 @@ import AverageRatings from './AverageRatings';
...
+  body {
+    font-family: Arial, Helvetica, sans-serif;
+    line-height: 1.2;
+    padding: 80px 0;
+    max-width: 1280px;
+    margin: 0 auto;
+    box-sizing: border-box;
+  }
+`;
+
```
Comment (JasperC1999) [inline client/components/App.jsx:30]: awesome!

File: client/components/App.jsx [hunk 2]
```diff
@@ -18,12 +22,17 @@ const Grid = styled.div`
   padding: 20px;
 `;
 
+const ReviewsTitle = styled.div`
+  text-transform: uppercase;
+  font-size: xx-large;
```
Comment (JasperC1999) [inline client/components/App.jsx:40]: what is xx-large?

File: client/components/App.jsx [hunk 3]
```diff
@@ -117,6 +126,12 @@ const App = (props) => {
   return (
     <div>
       <Grid>
+        <ReviewsTitle>Reviews</ReviewsTitle>
+        <div><ButtonRed buttonText="Write A Review" mouseClick={()=>{}} /></div>
```
Comment (JasperC1999) [inline client/components/App.jsx:144]: why do you need a mouseClick event listener here?

Comment (jayrbarr) [reply client/components/App.jsx:144]: You're right, I don't need it currently, but I was hoping for future implementation if I have time

File: client/components/App.jsx (hunk 2 above)
Comment (jayrbarr) [reply client/components/App.jsx:40]: It's one way to send in a responsive font size. I like the named conventions. Like with colors, too.

File: client/components/App.jsx (hunk 1 above)
Comment (jayrbarr) [reply client/components/App.jsx:30]: Thanks!

**Pull request merged**